### DISCUSSION TITLE
TARDIS Saves

### DISCUSSION THREAD

**Discussion by eccentricdevotion** [eccentricdevotion/TARDIS#397] [General]
#394 
Should TARDIS saves be associated with a TARDIS, or a Time Lord.
For example, if I enter someone else's TARDIS and open the TARDIS Saves GUI, should I see my own saves, or the saves of the owner of the TARDIS I'm in?

**Comment by Fristorm-Palifen** [comment]
I think it should be a good idea to make a switch for choosing if I want my save or other in the save gui ^^

**Reply by eccentricdevotion** [reply]
That’s a good idea :)

**Reply by eccentricdevotion** [reply]
7366a6b
